views/default/elements/typography.css.php

```diff
@@ -24,6 +24,25 @@
 	text-decoration: underline;
 }
 
+.elgg-anchor * {
+    display: inline;
+    vertical-align: middle;
+}
+
+.elgg-anchor * + .elgg-anchor-label {
+	margin-left: 5px;
+}
+
+.elgg-anchor-indicator:after {
+	content: attr(data-indicator);
+	display: inline-block;
+	margin-left: 5px;
+}
+
+.elgg-anchor-indicator[data-indicator="0"] {
+	display: none;
```

Comment:
The motivation is make it easier to update the counters with JS. You would just do `$('elem').attr('data-indicator', 5)`